bundles/org.openhab.binding.onewire/src/test/java/org/openhab/binding/onewire/device/DeviceTestParent.java

```diff
@@ -43,7 +43,7 @@
  * @author Jan N. Klug - Initial contribution
  */
 @NonNullByDefault
-public abstract class DeviceTestParent {
+public abstract class DeviceTestParent<T extends AbstractOwDevice> {
     protected @Nullable Class<? extends AbstractOwDevice> deviceTestClazz;
```

Comment:
If you make this as `Class<T>` you'll get rid of unchecked casting below.

Reply:
I think it can be also `private` now.

Reply:
done